### DISCUSSION TITLE
Welcome to react-boilerplate Discussions!

### DISCUSSION THREAD

**Discussion by Shchepotin** [brocoders/extensive-react-boilerplate#85] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KineBeo** [comment]
🚀 This is an incredibly great project for me! I have used your backend boilerplate in my projects, and it’s truly excellent.  
Currently, I’m starting to explore this boilerplate, so may I ask what the admin account is for accessing the admin panel?  

🙏 Thank you so much for your support!

**Reply by Shchepotin** [reply]
Hello @KineBeo ! You can find credentials here:

https://github.com/brocoders/nestjs-boilerplate/blob/4c95e496e1c419c05094f1f7c784dd33f38d1094/test/utils/constants.ts#L4-L5

**Reply by KineBeo** [reply]
Thank you so much <3
